src/cobald/daemon/config/mapping.py

```diff
@@ -96,15 +96,15 @@ def load_name(absolute_name: str):
             try:
                 obj = sys.modules[path[0]]
             except KeyError:
-                raise ImportError("No module named %r" % path[0])
+                raise ImportError("No module named %r" % path[0]) from None
```

Comment:
Should it not link to the "`key_error`" above?

Reply:
The ``KeyError`` is just an implementation detail here of modules being stored in a dict. I don't think it would add any relevant information, especially since the missing key is reported in the ``ImprotError`` as well.